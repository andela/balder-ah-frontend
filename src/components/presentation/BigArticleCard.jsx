import React from 'react';

const BigArticleCard = () => (
  <div className="col s12 m4">
    <div className="card hoverable">
      <div className="card-image">
        <img src="https://i.imgur.com/cKxsTfo.jpg" alt="smith" />
        <span className="card-title">Card Title</span>
      </div>
      <div className="card-content">
        <p>Brief text</p>
      </div>
      <div className="card-action">
        <a href="#!">Continue Reading</a>
      </div>
    </div>
  </div>
);

export default BigArticleCard;
