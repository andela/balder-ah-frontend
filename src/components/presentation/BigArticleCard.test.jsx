import React from 'react';
import { render, cleanup } from 'react-testing-library';
import 'jest-dom/extend-expect';
import BigArticleCard from './BigArticleCard';

afterEach(cleanup);

describe('<BigArticleCard />', () => {
  test('should render correctly', () => {
    const { container } = render(<BigArticleCard />);
    expect(container).toMatchSnapshot();
  });
});
