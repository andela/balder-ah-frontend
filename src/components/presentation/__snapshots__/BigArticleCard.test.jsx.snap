// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BigArticleCard /> should render correctly 1`] = `
<div>
  <div
    class="col s12 m4"
  >
    <div
      class="card hoverable"
    >
      <div
        class="card-image"
      >
        <img
          alt="smith"
          src="https://i.imgur.com/cKxsTfo.jpg"
        />
        <span
          class="card-title"
        >
          Card Title
        </span>
      </div>
      <div
        class="card-content"
      >
        <p>
          Brief text
        </p>
      </div>
      <div
        class="card-action"
      >
        <a
          href="#!"
        >
          Continue Reading
        </a>
      </div>
    </div>
  </div>
</div>
`;
